README-nightly.md

```diff
@@ -6,8 +6,8 @@
 
 A Python package for the mathematical modeling of infectious diseases via compartmental models. Originally designed for epidemiologists, epispot can be adapted for almost any type of modeling scenario.
 
-> This is a nightly version of epispot and may contain possibly unstable code.\
-> **Please see usage instructions prior to adding this project as a dependency**\
+> This is a nightly version of epispot and may contain possibly unstable code.  
```

Comment:
```suggestion
> This is a alpha version of epispot and may contain possibly unstable code.  
```

Reply:
While that's technically correct, it's a headache to maintain and not a meaningful change. There are some differences between nightly and alpha releases, but those differences are mentioned when appropriate. Here, differentiating between the two release types is unnecessary.